packages/cli/test/dev/fixtures/10a-nextjs-routes/pages/env.js

```diff
@@ -0,0 +1,14 @@
+function Env() {
+  return (
+    <main>
+      <p>
+        NEXT_PUBLIC_VERCEL_ENV: {process.env.NEXT_PUBLIC_VERCEL_ENV}
+      </p>
+      <p>
+        NEXT_PUBLIC_VERCEL_URL: {process.env.NEXT_PUBLIC_VERCEL_URL}
+      </p>
```

Comment:
If you want to make this test less dependent on react internals `<!-- -->`, you can use `<em>` instead.

And even add `<ul>` and `<li>`

```suggestion
      <ul>
      <li>
        NEXT_PUBLIC_VERCEL_ENV: <em>{process.env.NEXT_PUBLIC_VERCEL_ENV}</em>
      </p>
      <li>
        NEXT_PUBLIC_VERCEL_URL: <em>{process.env.NEXT_PUBLIC_VERCEL_URL}</em>
      </li>
      </ul>
```